src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -31,12 +31,18 @@ const AdContainer = styled.section`
   ${({ slotType }) => (slotType === 'mpu' ? mpuStyles : leaderboardStyles)}
 `;
 
-export const getBootsrapSrc = queryString => {
+export const getBootstrapSrc = queryString => {
```

Comment:
Is there potential for any odd behaviour regarding the querystring? I don't think mozart passes it through, so are we swapping out the script between server and client render?

Reply:
Does Mozart not pass additional query strings? It worked fine for the `adUnit` query string on test anyway

Reply:
Let's look into it separately from this issue :)